File: Py (3)/2.py
def promedio_notas():
    n = int(input("Ingrese la cantidad de notas: "))
    notas = []
    suma = 0

    for i in range(n):
        valor = float(input(f"Nota {i+1}: "))
        notas.append(valor)
        suma += valor

    promedio = suma / n
    print("El promedio de las notas es:", promedio)

promedio_notas()

File: C++(3)/2.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int n;
    cout << "Ingrese la cantidad de notas: ";
    cin >> n;

    vector<double> notas(n);
    double suma = 0;

    for (int i = 0; i < n; i++) {
        cout << "Nota " << i+1 << ": ";
        cin >> notas[i];
        suma += notas[i];
    }

    double promedio = suma / n;
    cout << "El promedio de las notas es: " << promedio << endl;

    return 0;
}
